### ISSUE TITLE
include/requre "File not found" message is too generic

### ISSUE THREAD

**Issue by fadimko** [facebook/hhvm#6684]
This message doesn't say to a programmer what caused the error, any function that works with files could have written this message. Also custom error handler, used in my company, can not parse it.

```
$ cat include-error.php
<?php
include ('noooo');
include 'noooo';
include_once ('noooo');
include_once 'noooo';
require_once ('noooo');
```
```
$ hhvm include-error.php 

Warning: File not found: noooo in /usr/local/src/hhvm/Include/include-error.php on line 2

Warning: File not found: noooo in /usr/local/src/hhvm/Include/include-error.php on line 3

Warning: File not found: noooo in /usr/local/src/hhvm/Include/include-error.php on line 4

Warning: File not found: noooo in /usr/local/src/hhvm/Include/include-error.php on line 5

Fatal error: File not found: noooo in /usr/local/src/hhvm/Include/include-error.php on line 6
```
```
$ php5 include-error.php 

Warning: include(noooo): failed to open stream: No such file or directory in /usr/local/src/hhvm/Include/include-error.php on line 2

Warning: include(): Failed opening 'noooo' for inclusion (include_path='.:/usr/local/share/pear') in /usr/local/src/hhvm/Include/include-error.php on line 2

Warning: include(noooo): failed to open stream: No such file or directory in /usr/local/src/hhvm/Include/include-error.php on line 3

Warning: include(): Failed opening 'noooo' for inclusion (include_path='.:/usr/local/share/pear') in /usr/local/src/hhvm/Include/include-error.php on line 3

Warning: include_once(noooo): failed to open stream: No such file or directory in /usr/local/src/hhvm/Include/include-error.php on line 4

Warning: include_once(): Failed opening 'noooo' for inclusion (include_path='.:/usr/local/share/pear') in /usr/local/src/hhvm/Include/include-error.php on line 4

Warning: include_once(noooo): failed to open stream: No such file or directory in /usr/local/src/hhvm/Include/include-error.php on line 5

Warning: include_once(): Failed opening 'noooo' for inclusion (include_path='.:/usr/local/share/pear') in /usr/local/src/hhvm/Include/include-error.php on line 5

Warning: require_once(noooo): failed to open stream: No such file or directory in /usr/local/src/hhvm/Include/include-error.php on line 6

Fatal error: require_once(): Failed opening required 'noooo' (include_path='.:/usr/local/share/pear') in /usr/local/src/hhvm/Include/include-error.php on line 6
```

**Comment by fredemmott** [comment]
Wishlist priority given that it includes the line number.

**Comment by fadimko** [comment]
If I provide a pr, will you review it?

**Comment by fredemmott** [comment]
Yep.

**Comment by fadimko** [comment]
#6686

Updated error messages:
```
$ hhvm include-error.php 

Warning: include(noooo): File not found in /usr/local/src/hhvm/Include/include-error.php on line 2

Warning: include(noooo): File not found in /usr/local/src/hhvm/Include/include-error.php on line 3

Warning: include_once(noooo): File not found in /usr/local/src/hhvm/Include/include-error.php on line 4

Warning: include_once(noooo): File not found in /usr/local/src/hhvm/Include/include-error.php on line 5

Fatal error: require_once(noooo): File not found in /usr/local/src/hhvm/Include/include-error.php on line 6
```

**Comment by fadimko** [comment]
fixed in 6cac021e4deb4a6dcd1ed772dd8bddee16061317
